juntagrico/entity/member.py

```diff
@@ -47,18 +47,24 @@ class Member(JuntagricoBaseModel):
     confirmed = models.BooleanField(_('E-Mail-Adresse bestätigt'), default=False)
     reachable_by_email = models.BooleanField(
         _('Kontaktierbar von der Job Seite aus'), default=False)
-
-    canceled = models.BooleanField(_('gekündigt'), default=False)
-    cancelation_date = models.DateField(
+    cancellation_date = models.DateField(
         _('Kündigüngsdatum'), null=True, blank=True)
+    deactivation_date = models.DateField(
+        _('Deaktivierungsdatum'), null=True, blank=True, help_text=_('Sperrt Login und entfernt von E-Mail-Listen'))
     end_date = models.DateField(
         _('Enddatum'), null=True, blank=True)
```

Comment:
add a help text, to clarify what this field means in comparison to the deactivation date.